config/geography/europe.go

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package geography
+
+const (
+	// Europe is the HCP deployment in Europe
+	Europe = "europe"
+)
+
+func init() {
+	Geographies = append(Geographies, Europe)
+}
+
+func NewConfigEurope() *Config {
+	return &Config{
+		APIAddress:        "api.cloud.eu.hashicorp.com",
+		AuthURL:           "auth.idp.eu.hashicorp.com",
```

Comment:
There URL is missing the scheme.

```suggestion
		AuthURL:           "https://auth.idp.eu.hashicorp.com",
```